app/src/main/java/com/example/ubclaunchpad/launchnote/edit/PhotoInfoActivity.kt

```diff
@@ -95,13 +95,26 @@ class PhotoInfoActivity : AppCompatActivity() {
         picNoteToEdit.title = title_input.text.toString()
         picNoteToEdit.description = description_input.text.toString()
         // todo vpineda we need to create a dropdown for all of the classes or projects!
-        if(folder_input.text.isNotEmpty())
+        if (folder_input.text.isNotEmpty()) {
             picNoteToEdit.folderId = Integer.parseInt(folder_input.text.toString())
```

Comment:
Are we going to only use integer names for folders? Or are we going to change this to strings?

Reply:
nvm, just saw the TODO